src/main/kotlin/ink/abb/pogo/scraper/tasks/CatchOneNearbyPokemon.kt

```diff
@@ -53,7 +53,8 @@ class CatchOneNearbyPokemon : Task {
 
                     if (result.status == CatchPokemonResponse.CatchStatus.CATCH_SUCCESS) {
                         ctx.pokemonStats.first.andIncrement
-                        var message = "Caught a ${catchablePokemon.pokemonId} with CP ${encounterResult.wildPokemon.pokemonData.cp} using $ball"
+                        val iv = (encounterResult.wildPokemon.pokemonData.individualAttack + encounterResult.wildPokemon.pokemonData.individualDefense + encounterResult.wildPokemon.pokemonData.individualStamina) * 100 / 45
```

Comment:
Yeah, it's another class, `pokemonData`. I spent ten minutes trying to find why it didn't let me use ivPercentage before figuring it out.